tls/s2n_handshake_io.c

```diff
@@ -635,6 +636,42 @@ static int handshake_read_io(struct s2n_connection *conn)
     return 0;
 }
 
+static void s2n_write_keylog(struct s2n_connection *conn)
+{
+       char *keylog_fn;
```

Comment:
your indentation is off, convention is 4 spaces, looks like you are using 7?

Reply:
Right. Sorry for not noticing myself. Fixed now.